renv/profiles/reporting/renv.lock

```diff
@@ -1712,15 +1712,15 @@
     },
     "terra": {
       "Package": "terra",
-      "Version": "1.7-83",
+      "Version": "1.8-10",
```

Comment:
Not sure why this only just started failing now ([example](https://github.com/ccao-data/model-res-avm/actions/runs/12776245674/job/35614885691#step:7:1023)) but it seems like `raster` is no longer installing properly. Updating its main dependency `terra` seems to fix the problem.